core/server/master/src/main/java/alluxio/master/file/meta/InodeView.java

```diff
@@ -103,12 +103,21 @@
   boolean isDeleted();
 
   /**
-   * @return true if the inode is a directory, false otherwise
+   * Checks whether the inode is a directory or not.
+   *
+   * @return  a boolean value representing whether the inode
+   *          is a directory. Returns {@code true} if it is
+   *          a directory, {@code false} otherwise.
    */
   boolean isDirectory();
 
   /**
-   * @return true if the inode is a file, false otherwise
+   * Checks whether the inode is a file.
+   *
```

Comment:
`Checks whether the inode is a file returning a boolean value indicating if it's or not`. What do you think?